.github/workflows/deployment.yml

```diff
@@ -0,0 +1,23 @@
+name: Django CI
+
+on:
+  push:
+    branches: [ develop, main ]
+  pull_request:
+    branches: [ develop, main ]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Build docker image
+      run: docker-compose up -d
+    - name: Run Tests
+      run: docker-compose run web python manage.py test
+    - uses: actions/setup-python@v2
```

Comment:
¡Bien! Probé en un enviroment local hecho con [Act](https://github.com/nektos/act) y funcionó la construcción de la imagen en docker y la ejecución de los test unitarios.

Sin embargo, eliminaría de esta línea para abajo. ¿Por qué? pre-commit/action se encuentra [deprecado](https://github.com/pre-commit/action) (Ver primeras líneas del readme). Me fue imposible probar esta action en mi entorno local. Para el CI de pre-commit, es recomendable seguir [estas instrucciones](https://pre-commit.ci/), que piden básicamente añadir algunas líneas al archivo `pre-commit-config.yaml` que ya tenemos. 
 
Adicionalmente, para que me funcionaran las actions tuve que modificar la versión presente en `docker-compose.yml` a `version: "2.4"`. Probé si esto afectaba la ejecución normal de nuestro proyecto y no pasó nada malo, pude buildear de nuevo la imagen normalmente, así que el downgrade no tiene ninguna consecuencia negativa. Añade también ese cambio aprovechando este PR.

Reply:
Habría que añadir esto en `pre-commit-config.yaml`:
`ci:
    autofix_commit_msg: |
        [pre-commit.ci] auto fixes from pre-commit.com hooks

        for more information, see https://pre-commit.ci
    autofix_prs: true
    autoupdate_branch: ''
    autoupdate_commit_msg: '[pre-commit.ci] pre-commit autoupdate'
    autoupdate_schedule: weekly
    skip: []
    submodules: false`

En teoría se debería respetar la configuración que ya tenemos para black, isort y flake8.

Reply:
@tfflores ahí hice los cambios que pediste